stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -190,3 +197,43 @@ internal fun DefaultComposerLabel() {
         color = ChatTheme.colors.textLowEmphasis
     )
 }
+
+/**
+ * Shows a [Toast] with an error if one of the following constraints are violated:
+ *
+ * - The message length exceeds the maximum allowed message length.
+ * - The number of selected attachments is too big.
+ * - At least one of the attachments is too big.
+ *
+ * @param validationErrors The list of validation errors for the current user input.
+ */
+@Composable
+private fun showValidationErrorIfNecessary(validationErrors: List<ValidationError>) {
```

Comment:
This is a simple solution, which is fine for now, but I'd really love it if we could do more work in the UI and e.g. show which attachments are too big or if the message is too long/has too many attachments somehow.